### PR TITLE
chore: refine the flow-unit-testing contribution document

### PR DESCRIPTION
**Pull request by taefi**

### CODE REVIEW

File: articles/guide/contributing/flow-unit-testing.asciidoc [hunk 1]
```diff
@@ -9,7 +9,36 @@ layout: page
...
+
+.Example standard names for unit test methods:
+[source]
+----
+void setValue_sameValue_firesNoEvent()
+void setValue_differentValue_firesOneEvent()
+----
+It is always a good practice to refer to existing tests as an example to how to write new tests.
+
+== How to Run the tests
```
Comment (taefi) [inline articles/guide/contributing/flow-unit-testing.asciidoc:36]:
```suggestion
== How to Run the Tests
```

File: articles/guide/contributing/flow-unit-testing.asciidoc [hunk 2]
```diff
@@ -9,7 +9,36 @@ layout: page
...
 
-.Work In Progress
-[NOTE]
-The unit testing instructions for Flow are in progress.
-In the meantime, you can explore existing unit tests in Flow project and use them as a basis for new tests.
+Every change in the code base requires a JUnit test for the code change.
+In cases where a JUnit test is not plausible then an <<flow-integration-testing#,Integration Test>> should be added instead.
+
+== Mocking
+JUnit mocks use Mockito and other mocking libraries should not be used as they may break when there are version updates.
```
Comment (taefi) [inline articles/guide/contributing/flow-unit-testing.asciidoc:16]:
```suggestion
JUnit mocks use `Mockito` and other mocking libraries should not be used as they may break when there are version updates.
```

**Review by mshabarov — APPROVED** [review]

**Comment by mshabarov** [comment]
How can I restart the `deploy/netlify`?

**Comment by jouni** [comment]
Try updating from upstream, there was a build problem earlier today which caused all the preview builds to fail, but it should be fixed now.

**Pull request merged**